src/serialmididevice.cpp

```diff
@@ -59,6 +60,18 @@ void CSerialMIDIDevice::Process (void)
 		return;
 	}
 
+	printf("Incoming MIDI data:\n");
```

Comment:
Let's do this only when `MIDIDumpEnabled=1` in `minidexed.ini`, since on lower-powered RPi models logging can cause audio issues.

Reply:
> Let's do this only when `MIDIDumpEnabled=1` in `minidexed.ini`, since on lower-powered RPi models logging can cause audio issues.

It's just fast debug code to find the problem... will be deleted later.